code/__HELPERS/roundend.dm

```diff
@@ -287,6 +287,9 @@
 	if(CONFIG_GET(flag/automapvote))
 		SSvote.initiate_vote("map", "BeeBot", forced=TRUE, popup=TRUE) //automatic map voting
 
+	//Save shuttles
+	INVOKE_ASYNC(SSorbits, /datum/controller/subsystem/processing/orbits.proc/save_custom_shuttles)
```

Comment:
Is this actually going to finish on a production server before the server reboots?

Reply:
Yes, there is 90 seconds after this is called before restarts, it finishes nearly instantly

Reply:
With 1024 files at 7kB it finished basically instantly.